vyxal/transpile.py

```diff
@@ -91,21 +91,20 @@ def transpile_token(
 
         # So instead, we have to manually escape the string
         string = f"{string!r}"
-        string = re.sub(r"\\([ntbrf])", r"\1", string).replace("\\`", "`")
+        string = re.sub(r"\\?\\([ntbrf])", r"\\\1", string).replace("\\`", "`")
```

Comment:
how many more times do you think we'll have to change this line before we decide to implement a proper string escaping system?

Reply:
At least 3